### ISSUE TITLE
[BadDomList] Needs to take drop time of dropped doms into account (Trac #1710)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1702]
If getting dropped doms from I3Live, it is still needed to verify the drop time. Otherwise, also dropped doms that are dropped after the (good) run end time are included.

Migrated from https://code.icecube.wisc.edu/ticket/1710
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:58", 
    "description": "If getting dropped doms from I3Live, it is still needed to verify the drop time. Otherwise, also dropped doms that are dropped after the (good) run end time are included.", 
    "reporter": "joertlin", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067178841456", 
    "component": "combo reconstruction", 
    "summary": "[BadDomList] Needs to take drop time of dropped doms into account", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2016-05-17T19:40:33", 
    "milestone": "", 
    "owner": "joertlin", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160518T17:19:05: 
* joertlin changed status from "new" to "closed"
* joertlin commented:

> In changeset 146061:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="146061"
> * Gets dark noise doms from i3live now
> * Checks if the drop time of a dom is before the good stop time. Closes #1702
> ```


* joertlin changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:58: 
* olivas changed milestone from "May 2016" to ""
* olivas commented:

> Milestone May 2016 deleted
